src/parser/rogue/outlaw/modules/talents/BladeRush.tsx

```diff
@@ -0,0 +1,57 @@
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Abilities from 'parser/core/modules/Abilities';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+import SPELLS from 'common/SPELLS';
+import Events, { CastEvent } from 'parser/core/Events';
+import RESOURCE_TYPES, { getResource } from 'game/RESOURCE_TYPES';
+
+class BladeRush extends Analyzer {
```

Comment:
Is this just an identical duplicate of the other module in outlaw/modules/spells? why?

Reply:
They are very similar. One is for Blade Flurry the other is for Blade Rush (talent)

Reply:
if they are almost identical, can they be combined? If not then everything i said about Blade Flurry needs to be done on this as well